### ISSUE TITLE
Layer-Neighbor Sampling

### ISSUE THREAD

**Issue by mfbalin** [gsampler9/gSampler#3]
I am the author and implementer of the https://docs.dgl.ai/generated/dgl.dataloading.LaborSampler.html, but I don't see a mention of it when it comes to discussing GPU-enabled graph sampling algorithms in your paper. And my PR was merged in November 2022. It supports full GPU acceleration and UVA sampling.

**Comment by Liu-rj** [comment]
Yes we have noticed your work but considering the papers was still on arXiv that time when this work is done, so we decided not to put it in our paper. However, we have discussed about the Layer-Neighbor Sampling and think actually it is a great consolidation to the motivation of gSampler's design since gSampler decouples the execution of layers but still provide holistic optimizations. We will look into your work and try to implement it using gSampler's framework and give you feedback one available. Thank your for mentioning this.:)

**Comment by Sunt-ing** [comment]
@mfbalin Hi, Muhammed. Your paper looks brilliant. Have you compared LABOR with KDD '21 PASS and Eurosys '21 Skywalker?
